app/src/main/res/menu/pdf_viewer.xml

```diff
@@ -37,6 +37,18 @@
         android:title="@string/action_zoom_in"
         app:showAsAction="ifRoom" />
 
+    <item
+        android:id="@+id/action_first"
+        android:icon="@drawable/ic_first_page_black_24dp"
```

Comment:
These should be the white icons rather than the black ones.

Reply:
Leftover from my old theme, now fixed.